build.gradle

```diff
@@ -36,8 +36,9 @@ buildscript {
         ktxVersion = '1.0.1'
         lifecycleVersion = '2.1.0-alpha02'
         materialVersion = '1.0.0'
-        navigationVersion = '1.0.0-rc01'
-        recyclerViewVersion = '1.1.0-alpha02'
+        navigationVersion = '1.0.0-rc02'
```

Comment:
Shall be changed?

master: navigationVersion = '2.0.0-rc02'

Reply:
This is OK for now.  Note that this PR will be merged into an `rv-selection` feature branch, which will be merged into `master` when ready